modules/administration_manual/pages/configuration/database/linux_database_configuration.adoc

```diff
@@ -155,14 +155,10 @@ The `config/config.php` as created by the xref:installation/installation_wizard.
 [[configure-mysql-for-4-byte-unicode-support]]
 ==== Configure MySQL for 4-byte Unicode Support
 
-For supporting such features as emoji, you have to enable 4-byte Unicode
-support in MySQL (instead of the default 3) _and_ in ownCloud. If you
-have a new installation, you don’t need to do anything, as mb4 support
-is checked during setup, and used if available. If it’s available,
-ownCloud is configured to use it.
+For supporting such features as emoji both MySQL (or MariaDB) *and* ownCloud need to be configured to use 4-byte Unicode support instead of the default 3-byte. 
+If you are setting up a new ownCloud installation, using version 10.0 or above, *and* you’re using a minimum MySQL version of 5.7, then you don’t need to do anything, as support is checked during setup and used if available. 
 
-However, if you have an existing installation that you need to convert
-to use 4-byte Unicode support, then you need to do two things:
+However, if you have an existing ownCloud installation that you need to convert to use 4-byte Unicode support or you are working with MySQL earlier than version 5.7, then you need to do two things:
 
 1.  In your MySQL configuration, add the configuration settings below.
```

Comment:
I think that there is a blank too much inbetween `1.` and `In your`

Reply:
Might well be. Thanks for spotting that one, @mmattel.